examples/IB/explicit/ex7/example.cpp

```diff
@@ -330,9 +343,24 @@ main(int argc, char* argv[])
         // Create Text File which Stores the locations of each IB point.
         std::ofstream fout;
 
+        // Define an MPI type for the Lag_Coords struct
+        const int items = NDIM + 1;
+
+        int blocklength[4] = { 1, 1, 1, 1 };
+        MPI_Datatype types[4] = { MPI_INT, MPI_DOUBLE, MPI_DOUBLE, MPI_DOUBLE };
+        MPI_Datatype mpi_lag_coords_type;
+        MPI_Aint offsets[4];
+        offsets[0] = offsetof(Lag_Coords, lag_idx);
+        offsets[1] = offsetof(Lag_Coords, X);
+        offsets[2] = offsetof(Lag_Coords, Y);
+        offsets[3] = offsetof(Lag_Coords, Z);
+
+        MPI_Type_create_struct(items, blocklength, offsets, types, &mpi_lag_coords_type);
```

Comment:
This needs a corresponding `MPI_Type_free()` call once we are done with it to prevent leaking memory.